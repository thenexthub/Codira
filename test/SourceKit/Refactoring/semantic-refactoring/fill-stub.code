protocol P {
  fn foo()
}

class C1 : P {}

// RUN: %empty-directory(%t.result)
// RUN: %sourcekitd-test -req=refactoring.fillstub -pos=5:8 %s -- %s > %t.result/fill-stub.code.expected
// RUN: %diff -u %S/fill-stub.code.expected %t.result/fill-stub.code.expected

// REQUIRES: OS=macosx || OS=linux-gnu
