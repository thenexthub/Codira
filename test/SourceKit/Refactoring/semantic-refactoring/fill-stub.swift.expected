source.edit.kind.active:
  5:15-5:15 "
    fn foo() {
        <#code#>
    }
"
